doc/release-notes/0.10/0.10.md

```diff
@@ -0,0 +1,187 @@
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/2533">#2533</a></td>
+<td valign="top">New OpenJ9 option, <code>-XX:[+|-]HeapDumpOnOutOfMemoryError</code>.</td>
+<td valign="top">OpenJDK8 and later: All platforms</td>
+<td valign="top">For compatibility with HotSpot, <code>-XX:[+|-]HeapDumpOnOutOfMemoryError</code> is accepted by OpenJ9, which can enable or disable dumps when an OutOfMemory exception occurs.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/2196">#2196</a></td>
+<td valign="top">New OpenJ9 option, <code>-XX:ActiveProcessorCount=<value></code>.</td>
+<td valign="top">OpenJDK8 and later: All platforms</td>
+<td valign="top">For compatibility with HotSpot, this option overrides the value for the number of CPUs that are automatically detected by the JVM. Additional Java dump information is recorded when the option is set.</td>
+</tr>
+
+</table>
+
+
+## Known Issues
+
+The v0.10.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support</td>
+<td valign="top">AIX</td>
+<td valign="top">Inability to diagnose problems with the VM, garbage collector, or JIT.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/54">#54</a></td>
+<td valign="top">Signal handling</td>
+<td valign="top">All platforms</td>
+<td valign="top">Currently, shutdown signals (<code>SIGINT</code>, <code>SIGHUP</code> and <code>SIGTERM</code>) and <code>SIGCONT</code> are fully supported on Unix platforms (pLinux, zLinux, xLinux, AIX and zOS). Full support for shutdown signals is pending on Windows; signal handlers cannot be registered by using <code>sun.misc.Signal</code> or <code>dk.internal.misc.Signal</code>. Also, support for other POSIX signals is pending. See <code>SunMiscSignalTest.java</code> for the list of signals that need to be supported.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/479">#479</a></td>
+<td valign="top">Non-compressed references VM builds not available</td>
+<td valign="top">AIX, Windows, Linux on Z, Linux on Power</td>
+<td valign="top">Non-compressed references VM builds, which support heaps larger than 57GB are supported on Linux on x86. Non-compressed references builds for other platforms are not available at AdoptOpenJDK.</td>
```

Comment:
I still don't like the wording. As @DanHeidinga pointed out elsewhere, "not available at AdoptOpenJDK" is an Adopt statement, not OpenJ9. From an OpenJ9 point of view, non-compressed refs is supported for all platforms, but only tested nightly for Linux x86.

Reply:
I'm wondering why we have this as a limitation at all. Perhaps I should just remove it? Or do you want a statement that we are not testing the other platforms? If someone builds it and finds a problem I'm guessing that we would help fix it.

Reply:
Its more like a FAQ than limitation but we could just say "Non-compressed references VM builds, which support heaps larger than 57GB, are built on Linux on x86." Which keeps the info about manual builds for other platforms.